scripts/cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Run this script regularly depending on disk usage
+./remove.sh
+rm -r /var/tmp/runner/_work/*
+sudo docker system prune -a --volumes
+
```

Comment:
It's quite probable that we'll want to upgrade to [docker-clean](https://github.com/ZZROTDesign/docker-clean) at some point, but we can leave this as a TODO for now